.github/workflows/libxrpl.yml

```diff
@@ -0,0 +1,85 @@
+name: Upload libXRPL to Conan
```

Comment:
What do you think of changing the name to something like "Check libXRPL compatibility with Clio"?

Reply:
Don't have to change it right now, but please let's all get in the habit of writing the library name in all lowercase: libxrpl. Unique capitalization is a frequent source of frustration when working with C++ libraries. CMake package names, and the variable and target names derived from them, are case-sensitive. It is easier for everyone if we all agree to use only lowercase. Who can remember "[NuDB](https://github.com/XRPLF/rippled/pull/4615/files)"?